src/ck-core/ckrdma.C

```diff
@@ -854,11 +854,10 @@ int getSrcPe(envelope *env) {
   PUP::fromMem up((void *)((CkMarshallMsg *)EnvToUsr(env))->msgBuf);
   up|numops;
   up|rootNode;
-  for(int i=0; i<numops; i++){
-    CkNcpyBuffer w;
-    up|w;
-    return w.pe;
-  }
+  CmiEnforce(numops > 0);
```

Comment:
Could be CmiAssert maybe

Reply:
Doesn't CmiAssert do nothing except for when built with '--enable-error-checking'?

Reply:
> Doesn't CmiAssert do nothing except for when built with '--enable-error-checking'?

Yes, I think so. I was just wondering if we should check the value of numops here in all build configs. If it's better with the check enabled, that's fine too.